### PR TITLE
updated READMEs

### LINKED ISSUE #23: Chore: Update Carbon package README
# Chore

## Description
New releases to npmjs.com can now be done successfully. However, [the package site documentation](https://www.npmjs.com/package/@cerner/carbon-graphs#getting-started) is pretty barebones and lacks information compared to before the migration. Therefore, `packages/carbon-graphs/README.md` needs to be updated so the documentation is up to date and has all the necessary information.

### PR DESCRIPTION
**Pull request by sdadn**
### Summary

Updated readme so that the Carbon package readme links to the rest of the documentation.

Closes #23

https://terra-graphs-deployed-pr-#.herokuapp.com/

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

### CODE REVIEW

**Review by mraplumb — APPROVED** [review]

File: packages/carbon-graphs/README.md [hunk 1]
```diff
@@ -1,20 +1,112 @@
...
+
+- [Installing Carbon](#installing-carbon)
 - [Getting Started](#getting-started)
-- [Documentation](../terra-graphs/docs/README.md)
+- [Graph Types](#graph-types)
 - [LICENSE](#license)
 
+## Installing Carbon
+
+- Install from [npmjs](https://www.npmjs.com/package/@cerner/carbon-graphs):
```
Comment (Veera81) [inline packages/carbon-graphs/README.md:18]: nit: Install from [npmjs](https://www.npmjs.com/package/@cerner/carbon-graphs) using the below command

File: packages/carbon-graphs/README.md [hunk 2]
```diff
@@ -1,20 +1,112 @@
...
 
+The data can now by plotted by as a line graph by using the following:
+
+```javascript
+    canvas.loadContent(Carbon.api.line(CONTENT_DATA1));
+```
+
+This results in the following graph:
+
+![Alt](../terra-graphs/docs/assets/carbon-simple-line-graph.png "simple line graph")
```
Comment (Veera81) [inline packages/carbon-graphs/README.md:89]: The image is not displaying

image

File: packages/carbon-graphs/README.md [hunk 3]
```diff
@@ -1,20 +1,112 @@
...
+-   [Spline Line](./packages/terra-graphs/docs/controls/Line.md#spline-line)
+-   [Paired Result](./packages/terra-graphs/docs/controls/PairedResult.md#usage)
+-   [Bar](./packages/terra-graphs/docs/controls/Bar.md#usage)
+-   [Timeline](./packages/terra-graphs/docs/controls/Timeline.md#usage)
+-   [Pie](./packages/terra-graphs/docs/controls/Pie.md#usage)
+-   [Gantt](./packages/terra-graphs/docs/controls/Gantt.md#usage)
+-   [Scatter](./packages/terra-graphs/docs/controls/Scatter.md#usage)
+-   [Bubble](./packages/terra-graphs/docs/controls/Bubble.md#usage)
+    -   [Bubble Single Dataset](./packages/terra-graphs/docs/controls/BubbleSingleDataset.md)
+    -   [Bubble Multiple Dataset](./packages/terra-graphs/docs/controls/BubbleMultipleDataset.md)
```
Comment (Veera81) [inline packages/carbon-graphs/README.md:107]: Are these links going to work once these changes are merged in the main branch?

I clicked on one of the links, it gave me the below error

image

**Review by Veera81 — CHANGES_REQUESTED** [review]

File: packages/carbon-graphs/README.md (hunk 2 above)
Comment (sdadn) [reply packages/carbon-graphs/README.md:89]: If you click on `view file`, the image should be working there:

CleanShot 2020-12-10 at 22 11 33@2x

CleanShot 2020-12-10 at 22 13 26@2x

File: packages/carbon-graphs/README.md (hunk 3 above)
Comment (sdadn) [reply packages/carbon-graphs/README.md]: Good catch, I fixed the paths here: 409361292dedfc8022adf4bdb36d18ca4d8ac015

File: packages/carbon-graphs/README.md (hunk 1 above)
Comment (sdadn) [reply packages/carbon-graphs/README.md]: Updated here: 9515cb45a727e543c1979448ea6c6255ec8d7328

**Pull request merged**